.github/workflows/ci.yml

```
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ['3.0', '3.1']
+        ruby: ['3.0', '3.1', '3.2.2']
```

```suggestion
        ruby: ['3.0', '3.1', '3.2']
```

Minor, but we commonly don't care too much about patch versions when it comes to the public repo test configs.